artifactory/services/utils/searchutil.go

```diff
@@ -243,6 +243,11 @@ func (item ResultItem) GetItemRelativePath() string {
 	return url
 }
 
+// Returns item.Repo/item.Path/
+func (item ResultItem) GetItemRelativeLocation() string {
+	return addSeparator(strings.ToLower(item.Repo), "/", strings.ToLower(item.Path)) + "/"
```

Comment:
Looks like the above expressions can be simplified to include only one *strings.ToLower* instead of two.